pkg/cli/interactive_tests/test_missing_log_output.tcl

```diff
@@ -63,6 +63,10 @@ eexpect ":/# "
 
 # Start a regular server
 send "$argv start >/dev/null 2>&1 & sleep 1\r"
+eexpect "\[1\]"
+eexpect ":/# "
+# Give the server some time to start.
+sleep 0.5
```

Comment:
We ought to have a mechanism that makes the server detach from the terminal, or indicates via a pipe/file its PID, only after it has started successfully. Unfortunately we don't have that.